src/pybind/mgr/nfs/module.py

```diff
@@ -50,6 +50,12 @@ def _cmd_nfs_export_create_cephfs(
                                              read_only=readonly, path=path,
                                              squash=squash, clients=clients)
 
+    @CLICommand('nfs export import', perm='rw')
```

Comment:
can we change this to `nfs export create`?. "import" would sound ok, if we'd be able to import some (e.g.) ceph-ansible exports this way.